app/src/pages/inside/logsPage/historyLine/historyLine.jsx

```diff
@@ -39,33 +55,67 @@ const cx = classNames.bind(styles);
   },
   { namespace: NAMESPACE },
 )
-@connect((state) => ({
-  historyItems: historyItemsSelector(state),
-  activeItemId: activeLogIdSelector(state),
-}))
+@connect(
+  (state) => ({
+    historyItems: historyItemsSelector(state),
+    activeItemId: activeLogIdSelector(state),
+  }),
+  {
+    fetchMoreItemsHistoryAction,
+  },
+)
+@injectIntl
 export class HistoryLine extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isLoading: false };
+  }
   static propTypes = {
+    intl: PropTypes.object.isRequired,
     historyItems: PropTypes.array,
     activeItemId: PropTypes.number,
     changeActiveItem: PropTypes.func,
+    fetchMoreItemsHistoryAction: PropTypes.func,
   };
 
   static defaultProps = {
     historyItems: [],
     activeItemId: 0,
     changeActiveItem: () => {},
+    fetchMoreItemsHistoryAction: () => {},
   };
 
   checkIfTheItemLinkIsActive = (item) =>
     item.id !== this.props.activeItemId && item.status !== NOT_FOUND;
 
-  render() {
-    const { historyItems, activeItemId, changeActiveItem } = this.props;
+  loadMoreItems = () => {
+    this.setState({ isLoading: true });
+    this.props.fetchMoreItemsHistoryAction(true);
+    this.setState({ isLoading: false });
```

Comment:
It looks like this `loading` will be stopped before the response is received from BE.
For the fast fix I suggest to use a `callback` in the action to set `isLoading: false`, when the response will be received.